docs/language/contract-updatability.md

```diff
@@ -225,6 +225,9 @@ A field may belong to a contract, struct, resource, or interface.
   pub struct Foo {
   }
   ```
+  This is because if there was a container of type `T` (i.e: an array `[T]` or dictionary `{R:T}`, etc.), 
+  and if a `Foo` value had been stored in that container, then that container value will become invalid,
+  as it will now hold a value that does not conform to `T`.
```

Comment:
Maybe also mention fields:

```suggestion
  This is because if there was a container of type `T` (e.g., an array `[T]` or dictionary `{R:T}`, etc.), or a field of type `T`,
  and if a `Foo` value had been stored in that container or field, then that container value will become invalid,
  as it will now hold a value that does not conform to `T`.
```

Reply:
added https://github.com/onflow/cadence-lang.org/pull/109